torch/testing/_internal/common_modules.py

```diff
@@ -2545,7 +2545,14 @@ def module_error_inputs_torch_nn_Pad3d(module_info, device, dtype, requires_grad
                module_inputs_func=module_inputs_torch_nn_AdaptiveAvgPool2d,
                skips=(
                    # Fails on MPS backend if input/output sizes are not divisible
-                   DecorateInfo(skipMPS),)
+                   DecorateInfo(skipMPS),
+                   # Fails on backward check if output size is 1x1
+                   DecorateInfo(
```

Comment:
Seems like the issue is that this gives an unexpected success when run using inductor. Do you know what the issue is here for 1x1 outputs for eager, would it be possible to fix it? 

Otherwise I am okay with skipping the test and filing an issue

Reply:
For 1x1 outputs, mean is applied on CPU instead of AdaptiveAvgPool2d so the grad will be channels first . Firstly, I added expectedFailure for cpu but it is still failed on CUDA. I will check this further.

Reply:
For unexpected success when run using inductor,  please see https://github.com/pytorch/pytorch/issues/107861

Reply:
Hm how come this doesn't give unexpected success in CI on this PR anymore?

Reply:
Oh thank you! I couldn't see the diff because it was force pushed, merging now